cpg_utils/cromwell.py

```diff
@@ -429,21 +445,38 @@ def _get_wait_interval(
     status_reported = False
     subprocess.check_output(GCLOUD_ACTIVATE_AUTH_BASE)  # noqa: S603
     cromwell_workflow_root = f'{CROMWELL_URL}/api/workflows/v1/{workflow_id}'
+    abort_url = f'{cromwell_workflow_root}/abort'
     metadata_url = f'{cromwell_workflow_root}/metadata'
     outputs_url = f'{cromwell_workflow_root}/outputs'
     status_url = f'{cromwell_workflow_root}/status'
     _remaining_exceptions = max_sequential_exception_count
     start = datetime.now()
+    cumulative_wait: int = 0
```

Comment:
Use a DateTime start here, and check time elapsed so you're not waiting for long requests.

Reply:
<img width="893" alt="Screenshot 2024-04-24 at 5 26 09 PM" src="https://github.com/populationgenomics/cpg-utils/assets/5886855/7935a5f4-1b7b-44c6-8727-bc50ffe0cb3b">
So rad that PR comments are now showing up in PyCharm